nixos/modules/services/networking/resilio.nix

```diff
@@ -81,105 +78,103 @@ in
         '';
       };
 
-      package = mkPackageOption pkgs "resilio-sync" { };
```

Comment:
Why is this removed?

Reply:
this was due to the conflicts. github shows just the removal. Due to rebase this should be fixed.